LongestPole/Unused.lean

```diff
@@ -0,0 +1,91 @@
+/-
+Copyright (c) 2024 Lean FRO. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kim Morrison
+-/
+import ImportGraph
+import Mathlib.Util.FormatTable
+import Batteries.Data.List.Basic
+import LongestPole.Rectangles
+import Cli
+
+/-!
+# `lake exe pole`
+
+Longest pole analysis for Mathlib build times.
+-/
```

Comment:
i see you copied the the initial contents from LongestPole/main.lean ... please update the module docs